doc/README.gpu_support

```diff
@@ -0,0 +1,128 @@
+# Open-CE GPU Support
+
+The Open-CE build scripts and feedstocks support CUDA from NVIDIA. In order
+for the build scripts to work, CUDA must be installed on the system on which
+the builds are taking place. This can be accomplished in two ways (see below).
+
+**In all cases, the `CUDA_HOME` environment variable must be set to the base
+directory where CUDA has been installed.**
+
+
+## Bare metal builds
+
+If you are using Open-CE on a bare metal system. Install the appropriate version
+of CUDA on the server that matches the version of Open-CE you are using. There
+is a reference table below.
+
+| Open-CE version | CUDA version |
+|-----------------|--------------|
+| 1.0 (marmotini) | 10.2         |
+|                 |              |
+
+It's important to install CUDA correctly. This procedure is dependent on the system
+type and CUDA version. It's advised to follow NVIDIA's [official installation documentation](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html)
+Note that NVLINK2-enabled POWER9 systems require a few [extra steps](https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#power9-setup)
+
+
+## Container builds
+
+It is recommended to run the Open-CE build scripts within a container. NVIDIA provides
+official CUDA base containers [on](https://hub.docker.com/r/nvidia/cuda) [dockerhub](https://hub.docker.com/r/nvidia/cuda-ppc64le),
+which are ideal for ensuring a properly installed CUDA environment. 
+
+Open-CE provides a working set of images that will work for running CUDA enabled builds.
+The dockerfiles are located in the [images directory](https://github.com/open-ce/open-ce/tree/master/images)
+These can be manually build using the command
+
+```shell
+docker build open-ce/images/build-cuda-x86_64
+```
+
+The Open-CE build image presets the `CUDA_HOME` environment variablet to the appropriate location.
+
+
+## Automatic docker builds
+
+The [build_env.py](https://github.com/open-ce/open-ce/blob/master/open-ce/build_env.py) script supports the `--docker_build` command line argument.
+This argument will automatically build the Open-CE CUDA-enabled build image and when combined 
+with the `--build_types=cuda` command line argument, it will build CUDA support into all of the 
+recipes included in the Open-CE environment file that are enabled to do so.
+
+
+## CUDA Build Type
+
+Both the [build_env.py](https://github.com/open-ce/open-ce/blob/master/open-ce/build_env.py) [build_feedstock.py](https://github.com/open-ce/open-ce/blob/master/open-ce/build_feedstock.py) scripts
+support the `--build_types=cuda` command line argument. This is required when CUDA support is desired in the build.
+This argument sets a parameter in the build environment can then be referenced in the
```

Comment:
This argument sets a parameter in the build environment *that/which* can then be referenced in the ....